docbook/Admin_Guide/en-US/config/misc.xml

```diff
@@ -121,6 +121,33 @@
 			</listitem>
 		</varlistentry>
 		<varlistentry>
+			<term>$g_public_urls</term>
+			<listitem>
+				<para>Enable usage of public share URLs. 
+					The default setting is OFF.
+				</para>
+				<para>Enabling this option will show additional "Public Share
+					URL" read-only field when viewing the bug (in "Users 
+					monitoring this issue" section) as well as additional 
+					"token" field in SOAP API. The URL can be copied, 
+					used and shared. 
+				</para>
+				<para>Access via public share URL does not require loggin in
+					(e.g. access is anonymous). This option does not 
+					require $g_allow_anonymous_login to be enabled.
+				</para>
+				<para>When user visits public share URL - he will see a limited
+					read-only view of the bug with "Full View" button asking
+					to log in. User may see current status of the bug and
+					minor details (such as modification date). Public bugs
+					will show summary, description and steps to reproduce in
+					limited view. Private bugs will show only summary, but no
```

Comment:
The concept of what to display is interesting.  I would rather if we don't add new rules and exceptions.  For example, for public issues, we should show all what a user with VIEWER access can see.  For private issue it is a bit tricky, since these are typically only visible for users with high access level like DEVELOPER, which means all issue details + private notes, etc would be visible.  The advantage of using the standard model of showing all what a user can see, is that it fits well with the configuration and business logic associated with all components that show up on the view issue page.  You can even re-use the same view issue page.  The "private issues" become the tricky part, we could exclude them from this, or think about some idea to cover them.